src/nv/storage/index.rs

```diff
@@ -61,31 +61,52 @@ bitfield! {
     pub struct NvIndexAttributes(TPMA_NV);
     impl Debug;
     // NV Index Attributes
-    pub pp_write, set_pp_write: 0;
-    pub owner_write, set_owner_write: 1;
-    pub auth_write, set_auth_write: 2;
-    pub policy_write, set_policy_write: 3;
+    pub pp_write, _: 0;
+    _, set_pp_write: 0;
```

Comment:
What was the problem with having the setter and the getter on the same line?

Reply:
For some weird reason I thought that the visibility specifier effected the line .. I will fix that.

Reply:
Makes sense, thanks!